login.html

```diff
@@ -1,45 +1,60 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>Teh Login Page</title>
-
-        <link rel="stylesheet" type="text/css" href="css/login.css">
-
-    </head>
-    <body>
-
-        <div id="external">
-            <div id="login">
-<!--                @Alex I'm not sure, but normally it should have an "action=login.php" or something similar attached-->
-                <h4>Interview Feedback</h4>
-                <form method="POST">
-                    <input type="text" name="userName" placeholder="User Name"> <br>
-                    <input type="password" name="pw" placeholder="Password"> <br>
-                    <input type="submit" name="login" value="Login">
-                </form>
-            </div>
-        </div>
-
-        <div class="feedback">
-            <img src="assets/new%20feedback.jpg" class="floated">
-            <h1 class="floated">Submit new feedback</h1>
-            <p class="floated">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
-            <div class="clearfix"></div>
-        </div>
-
 
-        <div class="feedback">
-            <img src="assets/results.jpg" class="floated2" >
-            <h1 class="floated2">View feedback results</h1>
-            <p class="floated2">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
-
-            <div class="clearfix"></div>
+<head>
+    <title>The Login Page</title>
+
+    <link rel="stylesheet/css" type="text/css" href="assets/css/normalize.css"/>
+    <link rel="stylesheet/css" type="text/css" href="assets/css/grid.less"/>
+    <link rel="stylesheet/less" type="text/css" href="assets/css/login.less">
+    <script src="less.js" type="text/javascript"></script>
+</head>
+
+<body>
+
+    <section class="login grid">
+        <div class="login-container">
```

Comment:
don't be afraid to apply 2-3-4 small, focused classes to the same element to achieve the desired effect